src/constants/node-env.ts

```diff
@@ -0,0 +1,19 @@
+export const LOCAL = 'local'
+export const DEVELOPMENT = 'development'
```

Comment:
This is interesting, because `NODE_ENV` is usually set to `development` when doing local development. I wonder what the best way to update scripts is for this?

This also gets into something that @zackkrida mentioned about the difference between NODE_ENV and the deployment stage. Like in our staging environment, `NODE_ENV` will be `production`, but maybe `STAGE_ENV` or some other custom variable we set, would be `staging`.

Should we rely on that instead maybe?

These are complicated!

I'm also wondering if we should encourage the use of `CI`, and `TEST` at all. In test presumably you would be testing specific flag combinations rather than some pre-set _single_ combination of flags (or at least we should be) and CI sort of conflicts with test in my head, I don't know when CI would be true but not test, at least when it comes to feature flags :thinking: 

Do you know if `NODE_ENV` is set to the string `'ci'` ever?

Reply:
I was also hesitant to use `NODE_ENV` for this purpose. It's very unclear when it's set to what and what it's possible values are. I'm not also sure about what else we could use.

Reply:
I think we can just create a new variable for it. `STAGE` is what our infrastructure uses to denote the "stage" of deployment, but I find it confusing with the word "staging" as one of those "stages" of deployment.

Could we use `DEPLOYMENT_ENV` and default it to `local`? Then we'd just need to update our staging and prod variables in our Terraform to define `DEPLOYMENT_ENV`. We also need this for Sentry as well.